_api-reference/cat/cat-shards.md

```diff
@@ -41,7 +41,7 @@ GET _cat/shards
 
 All cat shards URL parameters are optional.
 
-In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cat/index), you can specify the following parameter:
+In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-reference/cat/index), you can specify the following parameter:
```

Comment:
`[Compact and aligned text (CAT) API]({{site.url}}{{site.baseurl}}/api-reference/cat/index)`